tests/Image/Adapter/GmagickImageAdapterTest.php

```diff
@@ -11,7 +12,7 @@ class GmagickImageAdapterTest extends BaseImageAdapterTest
 {
     protected function getTestResourceInstance()
     {
-        return new Gmagick();
+        return new GMagick(__DIR__ . '/../../images/blank.png');
```

Comment:
This deserves an explanation, yes.

If `load()` is provided with an empty Gmagick object, the check for colorspace fails:

```
GmagickException: Can not process empty Gmagick object
```

Since the use of an empty Gmagick object is unlikely to occur in practice, I chose to modify the test case to use a non-empty object, rather than making special handling for empty objects in `load()`.

Let me add a comment.